scripts/ie11-styles-post-processing.js

```diff
@@ -0,0 +1,52 @@
+const presetEnv = require('postcss-preset-env');
+const postcss = require('postcss');
+const fs = require('fs');
+const fse = require('fs-extra');
+const cssvariables = require('postcss-css-variables');
+const path = require('path');
+const fromPath = '../node_modules/@patternfly/patternfly-next/patternfly.css';
```

Comment:
Something that might throw a wrench in this is the injection of react-core styles at consumption time. It may cause issues with the overrides from here and the ie11.css. Best bet would be to test with the grid component on IE11. The injected CSS from react-core will not get run through the post CSS plugin so all the variable references will remain too.

Reply:
Ah, good point. Yea grid is still not 100% (but it's getting close) so I'll try to get that working and then will test how they play with the injected css from react-core. Good call!

Reply:
So, managed to get grid going successfully, but as @dmiller9911 mentioned this breaks down when relying on react components to pull in the styles. Progress none-the-less!

I need to spend some time looking into how the styles are managed with the react components, maybe there's a way to have them instead fetch the ie-compatible versions of these styles? Or maybe there's a way to tell the react components to not inject styles at all, and rather just operate the markup and allow styles to be received from the document in a more traditional manner? Seems like the latter should be possible..

Let me know if either of these options sound better than the other, I'll press ahead in the mean time.